transcom/mymove
# B-20281 SC defects

## [Agility ticket B-20281](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A972029)
## [Agility ticket B-20284](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A972070&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## [INT PR](https://github.com/transcom/mymove/pull/13290)

## Summary

There were 2 defects found in the current code in main.
1. A SC is able to submit a MTO without any documents uploaded.
2. The Allowances card shows 0 for weight for a SC.

### How to test

1. Login as a SC.
2. Find a move in 'Needs counseling' status for that SC.
3. Fill in the required orders fields (Dept Indicators, Orders number, etc).
4. Click on Manage Orders and delete any orders currently attached.
5. Click Save and verify that the Submit move details button is disabled.
6. Go back and upload an Order, verify that Submit is now enabled.
7. Another approach is to create a new move as a SC.  Verify that the Submit move details button is not enabled until you have everything filled out and an order uploaded.

1.  On a new or existing move from test steps above, verify that the Weight allowance field in the Allowances card on move details has the correct value for the SC and is not showing 0.
